### PR TITLE
Upgrade seed plugin

### PR DESCRIPTION
**Pull request by fedgehog**

### CODE REVIEW

File: src/main/scala/com/thenewmotion/akka/rabbitmq/ConnectionActor.scala [hunk 1]
```diff
@@ -58,18 +58,18 @@ class ConnectionActor(
...
+    case Event(ProvideChannel, _) =>
+      log.debug("{} can't create channel for {} in disconnected state", header(Disconnected, ProvideChannel), sender())
       stay()
   }
   when(Connected) {
-    case Event(msg @ ProvideChannel, Connected(connection)) =>
+    case Event(ProvideChannel, Connected(connection)) =>
       safe(connection.createChannel()) match {
         case Some(channel) =>
-          log.debug("{} channel acquired", header(Connected, msg))
```
Comment (t3hnar) [inline src/main/scala/com/thenewmotion/akka/rabbitmq/ConnectionActor.scala]: not sure this is an improvement

Comment (fedgehog) [inline src/main/scala/com/thenewmotion/akka/rabbitmq/ConnectionActor.scala]: According to compiler it is. If you checkout commit right before that. Compiler will give this nice warning
/.../akka-rabbitmq/src/main/scala/com/thenewmotion/akka/rabbitmq/ConnectionActor.scala:61: The value matched by ConnectionActor.ProvideChannel is bound to msg, which may be used under the unsound assumption that it has type com.thenewmotion.akka.rabbitmq.ConnectionActor.ProvideChannel.type, whereas we can only safely count on it having type Any, as the pattern is matched using `==` (see SI-1503).

Comment (t3hnar) [inline src/main/scala/com/thenewmotion/akka/rabbitmq/ConnectionActor.scala]: thx @fedgehog, did not know that.

**Comment by t3hnar** [comment]
LGTM

File: src/main/scala/com/thenewmotion/akka/rabbitmq/ConnectionActor.scala (hunk 1 above)
Comment (fedgehog) [inline src/main/scala/com/thenewmotion/akka/rabbitmq/ConnectionActor.scala]: I agree that it looks silly and would expect it to work fine. Still if we can remove easily avoidable compiler warnings that would help against warning blindness in future for more important cases.

Comment (t3hnar) [inline src/main/scala/com/thenewmotion/akka/rabbitmq/ConnectionActor.scala]: totally agree

**Pull request merged**